svc-lighthouse-api/src/test/java/gov/va/vro/abddataaccess/service/FhirClientTest.java

```diff
@@ -50,6 +48,31 @@
 @Slf4j
 @ActiveProfiles("test")
 class FhirClientTest {
+  @AllArgsConstructor
+  @Getter
+  private static class BundleInfo {
+    private Bundle bundle;
+    private String responseBody;
+
+    private static Bundle getMedicalInfoBundle(String filename)
+        throws FileNotFoundException, IOException {
```

Comment:
Will the exceptions be handled? What's the effect when the exception happened?

Reply:
This is a test. If exceptions happen the test will fail.